Packs/iDefense/Integrations/iDefense_v2/iDefense_v2.yml

```diff
@@ -13,6 +13,20 @@ configuration:
   defaultvalue: ""
   type: 4
   required: true
+- additionalinfo: Reliability of the source providing the intelligence data.
+  defaultvalue: B - Usually reliable
+  display: Source Reliability
+  name: integrationReliability
+  options:
+  - A+ - 3rd party enrichment
+  - A - Completely reliable
+  - B - Usually reliable
+  - C - Fairly reliable
+  - D - Not usually reliable
+  - E - Unreliable
+  - F - Reliability cannot be judged
+  required: true
```

Comment:
it has a defaultvalue, so i do not think it should be required.
@yuvalbenshalom WDYT?